src/components/MapEditor/__tests__/MapInteractive.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Map from '..';
+import MAP_OPTIONS from '../../../shared/services/configuration/map-options';
+
+describe('components/MapInteractive', () => {
+  const testLocation = {
+    geometrie: {
+      type: 'Point',
+      coordinates: [4, 52],
+    },
+  };
+
+  it('should render the map', () => {
+    const { container, getByText } = render(<Map options={MAP_OPTIONS} location={{}}/>);
+
+    // Map
+    expect(getByText('Leaflet')).toBeInTheDocument();
+
+    // Tile layer
+    expect(container.querySelector('.leaflet-tile-container')).toBeInTheDocument();
+
+    // Zoom
+    expect(container.querySelector('button[title="Inzoomen"]')).toBeInTheDocument();
+    expect(container.querySelector('button[title="Uitzoomen"]')).toBeInTheDocument();
```

Comment:
The test breaks on this assertion. Check the Jenkins console output for this PR's build.

Reply:
I have corrected this in the amsterdam-react-maps, there was the text wrong for the Zoom out button.